x/posts/internal/types/keys.go

```diff
@@ -17,18 +17,24 @@ const (
 	LastPostIDStoreKey       = "last_post_id"
 	PostCommentsStorePrefix  = "comments:"
 	PostReactionsStorePrefix = "reactions:"
+	PollAnswersStorePrefix   = "poll_answers:"
 
 	MaxPostMessageLength            = 500
 	MaxOptionalDataFieldsNumber     = 10
 	MaxOptionalDataFieldValueLength = 200
 
 	ActionCreatePost         = "create_post"
 	ActionEditPost           = "edit_post"
+	ActionClosePollPost      = "close_poll_post"
```

Comment:
I think we can remove the trailing `post` here

```suggestion
	ActionClosePollPost      = "close_poll"
```